app/kumactl/cmd/install/install_transparent_proxy.go

```diff
@@ -168,10 +168,11 @@ runuser -u kuma-dp -- \
 				}
 			}
 
+			defaultCfg := config.DefaultConfig()
 			// Backward compatibility
 			if args.RedirectPortInBoundV6 != "" &&
-				args.RedirectPortInBoundV6 != "15006" /* new default value, identical to ipv4 port */ &&
-				args.RedirectPortInBoundV6 != "15010" /* old default value, dedicated for ipv6 */ {
+				args.RedirectPortInBoundV6 != fmt.Sprintf("%d", defaultCfg.Redirect.Inbound.Port) /* new default value, identical to ipv4 port */ &&
```

Comment:
how is sprintf better here?

Reply:
On, it's not Sprintf, but we read the value from `defaultCfg`, instead of a literal.